ERCS/erc-7688.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: 7688
+title: Ethereum RiskWatch Guide
```

Comment:
```suggestion
title: Transaction risk monitoring
```
Suggest using a more descriptive title.  This is an ERC so doesn't need to include the word Ethereum.